src/components/asset-amount-input/asset-amount-input.tsx

```diff
@@ -93,7 +98,13 @@ const AssetAmountInputComponent: FC<AssetAmountInputProps> = ({
           return mutezToTz(value.amount, value.asset.decimals);
         } else {
           if (isDefined(inputValueRef.current)) {
-            const currentTokenValue = dollarToTokenAmount(inputValueRef.current, value.asset.decimals, exchangeRate);
+            const fiatToUsdRate = quotes[fiatCurrency.toLowerCase()] / exchangeRateTezos;
```

Comment:
create `useFiatToUsdRate` hook that will return this value

Reply:
it is better to create `useFiatExchangeRateGetter` hook that will return fn `(asset) =>  fiatExhangeRate`